daemon/api_notices_test.go

```diff
@@ -339,17 +365,17 @@ func (s *noticesSuite) TestNoticesFilterTypesForSnapForbidden(c *C) {
 	rsp = s.errorReq(c, req, nil)
 	c.Check(rsp.Status, Equals, 403)
 
-	// snap-themes-control doesn't give access to change-update notices.
+	// snap-prompting-control doesn't give access to change-update notices.
 	req, err = http.NewRequest("GET", "/v2/notices?types=change-update", nil)
 	c.Assert(err, IsNil)
-	req.RemoteAddr = fmt.Sprintf("pid=100;uid=1000;socket=%s;iface=snap-themes-control;", dirs.SnapSocket)
+	req.RemoteAddr = fmt.Sprintf("pid=100;uid=1000;socket=%s;iface=snap-prompting-control;", dirs.SnapSocket)
 	rsp = s.errorReq(c, req, nil)
 	c.Check(rsp.Status, Equals, 403)
 
-	// snap-themes-control doesn't give access to refresh-inhibit notices.
+	// snap-prompting-control doesn't give access to refresh-inhibit notices.
 	req, err = http.NewRequest("GET", "/v2/notices?types=refresh-inhibit", nil)
 	c.Assert(err, IsNil)
-	req.RemoteAddr = fmt.Sprintf("pid=100;uid=1000;socket=%s;iface=snap-themes-control;", dirs.SnapSocket)
+	req.RemoteAddr = fmt.Sprintf("pid=100;uid=1000;socket=%s;iface=snap-prompting-control;", dirs.SnapSocket)
```

Comment:
these changes are a bit strange, what I think would make sense it to parametrize the test and run both the old variant and again with the new interface

Reply:
Yes, will do!